recipe/meta.yaml

```diff
@@ -19,6 +19,9 @@ build:
   rpaths:
     - lib/R/lib/
     - lib/
+  ignore_run_exports_from:
+    # TODO: See below. Remove explicit dependency when workaround is not needed anymore.
+    - openssl <3.2.0a0               # [not win]
```

Comment:
This is mostly just pedantry.
The added `openssl >=3.1,<4.0a0` constraint is given indirectly via `libcurl` so wouldn't hurt to be added.
The metadata is just "cleaner" if we don't add it.